client/src/main/java/org/apache/rocketmq/client/consumer/DefaultLitePullConsumer.java

```diff
@@ -188,6 +200,57 @@ public DefaultLitePullConsumer(final String consumerGroup, RPCHook rpcHook) {
         this(null, consumerGroup, rpcHook);
     }
 
+    /**
+     * Constructor specifying consumer group and enabled msg trace flag.
+     *
+     * @param consumerGroup  Consumer group.
+     * @param enableMsgTrace Switch flag instance for message trace.
+     */
+    public DefaultLitePullConsumer(final String consumerGroup, boolean enableMsgTrace) {
+        this(null, consumerGroup, null, enableMsgTrace, null);
+    }
+
+    /**
+     * Constructor specifying consumer group, enabled msg trace flag and customized trace topic name.
+     *
+     * @param consumerGroup        Consumer group.
+     * @param enableMsgTrace       Switch flag instance for message trace.
+     * @param customizedTraceTopic The name value of message trace topic.If you don't config,you can use the default
+     *                             trace topic name.
+     */
+    public DefaultLitePullConsumer(final String consumerGroup, boolean enableMsgTrace,
```

Comment:
How about removing this constructor? Maybe use the set method to set related configurations would be a better choice.

Reply:
This is a good suggestion, but this submission still wants to be consistent with DefaultMQPushConsumer and DefaultMQProducer. If necessary, a PR can be proposed later to rectify the problem of too many constructors